core/domain/exp_services.py

```diff
@@ -1610,18 +1611,22 @@ def regenerate_exploration_and_contributors_summaries(exploration_id):
 
 
 def compute_summary_of_exploration(
-    exploration, exp_rights, exp_summary_model,
-    skip_exploration_model_last_updated=False):
+    exploration,
+    exp_rights,
+    exp_summary_model,
```

Comment:
Shouldn't be passing models around, as mentioned before. Can we use domain objects instead?

For Beam: can Beam get the models, convert it to domain objects outside this function, and then pass those domain objects into this function instead? Or have a wrapper in the Beam jobs class that uses this function internally and does the model conversions there? I am concerned about making this API (with model-type arguments) general-use in this file.

Reply:
Done.